### DISCUSSION TITLE
Select a curation before inserting the image in TipTap Editor

### DISCUSSION THREAD

**Discussion by christopher87** [awcodes/filament-curator#543] [Q&A] [answered]
Hi,

Is it possible to select a curation (pre-defined size of the image) before inserting in the WYSIWYG TipTap Editor instead of the full size image ?

For example in WordPress, you can choose from the modal the pre-defined size of the image you want to insert :
![image](https://github.com/user-attachments/assets/a5966ac3-4cbf-424b-9827-371f98843fed)

Thanks in advance,

**Comment by awcodes** [comment, marked as answer]
Currently there is not. I'll make a note of it. Not sure when I can get to it though. But would be a good PR.

**Reply by christopher87** [reply]
Thank you for your quick answer :)
